internal/cmd/egctl/config.go

```diff
@@ -180,8 +180,14 @@ func fetchRunningEnvoyPods(c kube.CLIClient, nn types.NamespacedName, labelSelec
 }
 
 // portForwarder returns a port forwarder instance for a single Pod.
-func portForwarder(cli kube.CLIClient, nn types.NamespacedName) (kube.PortForwarder, error) {
-	fw, err := kube.NewLocalPortForwarder(cli, nn, 0, adminPort)
+func portForwarder(cli kube.CLIClient, nn types.NamespacedName, port ...int) (kube.PortForwarder, error) {
+	var err error
+	var fw kube.PortForwarder
+	if len(port) == 0 {
+		fw, err = kube.NewLocalPortForwarder(cli, nn, 0, adminPort)
+	} else {
+		fw, err = kube.NewLocalPortForwarder(cli, nn, 0, port[0])
```

Comment:
if only port[0] is used, why do we need a slice here?

Reply:
It was more or less a way to provide default port if no values is present. I can make it an `int` if it makes more sense